src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -178,6 +180,7 @@
       id="{{ _id }}-label"
     {% endif %}
       class="{{- _label_class -}}
+      {{- _hide_label ? ' ecl-u-sr-only' -}}
```

Comment:
mmh..fine eh, but i would stick to the idea of not using directly utilities inside the components markup, otherwise in a way we are also "accepting" this practice by our users.
We can add a selector with a semantic name and extend the ecl-u-sr-only in the component's sass file.

Reply:
indeed. That's how we do it in the buttons already, but I guess that it could be updated there too in the future

Reply:
(not the button, but the rating field)